content/gocourse-2019-05-melbourne.slide

```diff
@@ -841,31 +841,18 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
 - Bonus points: Add a third `Storer` implementation using [[https://github.com/syndtr/goleveldb][LevelDB]]
 
 
-* Concurrency (1/2)
+* Go Concurrency
 
-"Don't communicate by sharing memory, share memory by communicating."
```

Comment:
I feel like these first four dot-points still have a place here
They talk about the mindset the creators had going into developing go's concurrency model.

IMO add a slide after the concurrency title slide and before the topics.
slide title suggestion: 'Concurrency in go'

Reply:
Yup, I agree with that.

Reply:
I'll do that

Reply:
Done